src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be tagged.
+- One or more tags can be added to the specific **thread**.
+- To tag a chat, the **tag** needs to be created first. It can be done in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api).
+- Tags are case sensitive.
+- Tags are defined by each client individually. The **list of tags** can be found in the [LiveChat Application](https://my.livechatinc.com/settings/tags). 
+   
+The **tagged threads** can be untagged using [Untag Thread](#untag-thread).
```

Comment:
I don't think we mention opposite methods in other places (eg remove_user in add_user description) 🤔 

anyway, if we are going to do so, let's go with "to remove an existing tag from the thread use..."
"tagged thread can be untagged" sounds... meh

Reply:
It is not crucial, so I will remove it :)